src/utils/kubernetes/kubernetes-routes.js

```diff
@@ -0,0 +1,211 @@
+        logger.error("Error getting httproutes: %d %s %s", error.statusCode, error.body, error.response);
+        logger.debug(error);
+        return null;
+      });
+
+  // namespaces
+  const namespaces = await core
+    .listNamespace()
+    .then((response) => response.body.items.map((ns) => ns.metadata.name))
+    .catch((error) => {
+      logger.error("Error getting namespaces: %d %s %s", error.statusCode, error.body, error.response);
+      logger.debug(error);
+      return null;
+    });
+
+  let httpRouteList = [];
+  if (namespaces) {
+    const httpRouteListUnfiltered = await Promise.all(
+      namespaces.map(async (namespace) => {
+        const httpRoute = await getHttpRoute(namespace);
+        return httpRoute;
+      }),
+    );
+
+    httpRouteList = httpRouteListUnfiltered.filter((httpRoute) => httpRoute !== undefined);
+  }
+  return httpRouteList;
+}
+
+async function getIngressList(ANNOTATION_BASE) {
+  const ingressList = await networking
+    .listIngressForAllNamespaces(null, null, null, null)
+    .then((response) => response.body)
+    .catch((error) => {
+      logger.error("Error getting ingresses: %d %s %s", error.statusCode, error.body, error.response);
+      logger.debug(error);
+      return null;
+    });
+
+  if (traefik) {
+    const traefikContainoExists = await checkCRD("ingressroutes.traefik.containo.us");
+    const traefikExists = await checkCRD("ingressroutes.traefik.io");
+
+    const traefikIngressListContaino = await crd
+      .listClusterCustomObject("traefik.containo.us", "v1alpha1", "ingressroutes")
+      .then((response) => response.body)
+      .catch(async (error) => {
+        if (traefikContainoExists) {
+          logger.error(
+            "Error getting traefik ingresses from traefik.containo.us: %d %s %s",
+            error.statusCode,
+            error.body,
+            error.response,
+          );
+          logger.debug(error);
+        }
+
+        return [];
+      });
+
+    const traefikIngressListIo = await crd
+      .listClusterCustomObject("traefik.io", "v1alpha1", "ingressroutes")
+      .then((response) => response.body)
+      .catch(async (error) => {
+        if (traefikExists) {
+          logger.error(
+            "Error getting traefik ingresses from traefik.io: %d %s %s",
+            error.statusCode,
+            error.body,
+            error.response,
+          );
+          logger.debug(error);
+        }
+
+        return [];
+      });
+
+    const traefikIngressList = [...(traefikIngressListContaino?.items ?? []), ...(traefikIngressListIo?.items ?? [])];
+
+    if (traefikIngressList.length > 0) {
+      const traefikServices = traefikIngressList.filter(
+        (ingress) => ingress.metadata.annotations && ingress.metadata.annotations[`${ANNOTATION_BASE}/href`],
+      );
+      ingressList.items.push(...traefikServices);
+    }
+  }
+
+  return ingressList.items;
+}
+
+export async function getRouteList(ANNOTATION_BASE) {
```

Comment:
```suggestion
export async function getRouteList(ANNOTATION_BASE) {
```

Same as above, do not write method parameters like this